src/main/java/com/camunda/consulting/simulator/jobhandler/ClaimUserTaskJobHandler.java

```diff
@@ -0,0 +1,73 @@
+package com.camunda.consulting.simulator.jobhandler;
+
+import org.camunda.bpm.engine.ProcessEngineException;
+import org.camunda.bpm.engine.impl.interceptor.CommandContext;
+import org.camunda.bpm.engine.impl.jobexecutor.JobHandler;
+import org.camunda.bpm.engine.impl.jobexecutor.JobHandlerConfiguration;
+import org.camunda.bpm.engine.impl.jobexecutor.TimerEventJobHandler;
+import org.camunda.bpm.engine.impl.persistence.entity.ExecutionEntity;
+import org.camunda.bpm.engine.impl.persistence.entity.JobEntity;
+
+public class ClaimUserTaskJobHandler implements JobHandler<ClaimUserTaskJobHandler.ClaimUserTaskJobHandlerConfiguration> {
+
+  public static final String TYPE = "simulateClaimUserTask";
+
+  @Override
+  public String getType() {
+    return TYPE;
+  }
+
+  @Override
+  public void execute(ClaimUserTaskJobHandlerConfiguration configuration, ExecutionEntity execution, CommandContext commandContext, String tenantId) {
+    String taskId = configuration.getTaskId();
+    String userId = configuration.getUserId();
+    // eventually we created a claim job, but before it gets executed the usertask is completed
```

Comment:
This check is not necessary. When the task is completed, all timer jobs that belong to it will be removed, including the claim job.

Reply:
In fact, this is only true if the user task is a scope. So we force them to be scope which should not cause any harm but prevents us from writing timer job clean-up code. Done in 97d3103700669d0006955d392248f88d159bc0c5.